### PR TITLE
Home

### PR DESCRIPTION
**Pull request by Digambar1411**
Added home page

Components
-footer
-navbar
-sidebar

Contexts
-theme-context
-sidebar-context

### CODE REVIEW

**Review by simran61 — APPROVED** [review]

File: src/App.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import "./App.css";
+// import "./App.css";
 import {Routes, Route} from "react-router-dom";
 import {Home, LikedVideos, WatchLater, Login, Signup, VideoListing} from "../src/frontend/pages/index";
```
Comment (omkarpatke) [inline src/App.js:3]: appreciation: good manner of importing files.

File: src/frontend/components/navbar/nav.css [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+    height: 5rem;
+    position: fixed;
+    top: 0;
+    align-items: center;
+    z-index: 3;
+    background-color: var(--secondary-background-color); 
+}
+
+.brand{
+    font-size: 1.75em;
```
Comment (omkarpatke) [inline src/frontend/components/navbar/nav.css]: suggestion: use rem instead of em

File: src/frontend/components/navbar/nav.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
+import "./nav.css";
+import { useSidebar } from "../../contexts/index";
+import { useTheme } from "../../contexts/index";
```
Comment (omkarpatke) [inline src/frontend/components/navbar/nav.js]: suggestion: both imports have the same path so you can import it from one.

**Review by omkarpatke — APPROVED** [review]
good folder structure also good use of CSS variables did import-export in a good manner do suggested changes and you are good to go.

File: src/frontend/components/navbar/nav.js (hunk 3 above)
Comment (Digambar1411) [reply src/frontend/components/navbar/nav.js]: Yeah right  thanks for pointing out I will do it

File: src/App.js (hunk 1 above)
Comment (Digambar1411) [reply src/App.js:3]: thanks

**Pull request merged**